sandbox/src/routes/retrieveAdviceAndGuidanceRequest.js

```diff
@@ -0,0 +1,38 @@
+const mockResponseProvider = require('../services/mockResponseProvider')
+const businessFunctionValidator = require('../services/businessFunctionValidator')
+
+function retrieveAdviceAndGuidanceRequest(request, h) {
+  const allowedBusinessFunctions = ["SERVICE_PROVIDER_CLINICIAN"]
```

Comment:
Should this include more roles?

Reply:
That was updated after I've done the initial development :) RC and RCA will be added here but on my next ticket: ERSSUP-54588